qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -68,3 +68,8 @@
                     start_vm = no
                     check_die_id = 'cat /sys/devices/system/cpu/cpu*/topology/die_id | sort | uniq -c'
                     check_die_cpus_list = 'cat /sys/devices/system/cpu/cpu*/topology/die_cpus_list | sort | uniq -c'
+                - nonstop_tsc:
+                    type = x86_cpu_flag_nonstop_tsc
+                    check_clock = 'cat /sys/devices/system/clocksource/clocksource0/available_clocksource | grep tsc'
+                    cpu_model_flags += ",+invtsc"
```

Comment:
Would you pls check since when this flag became supported?

Reply:
I mean to add qemu restriction.

Reply:
> I mean to add qemu restriction.

we have a related FeatureFeature bug on RHEL7.1， so I think it should cover all version qemu.

Reply:
Make sense.